app/retail/templates/ideas_list.html

```diff
@@ -0,0 +1,63 @@
+{% comment %} Copyright (C) 2017 Gitcoin Core This program is free software: you can redistribute it and/or modify it under
```

Comment:
Can we resolve the comment block here and move includes and what not to their own lines?